test/node.js

```diff
@@ -604,7 +608,13 @@ node.initApplication = function (cb, initScope) {
 			scope.modules.delegates.onBlockchainReady = function () {};
 			scope.rewiredModules = rewiredModules;
 			currentAppScope = scope;
-			cb(err, scope);
+			if (initScope.noWaitForGenesisBlock) {
```

Comment:
this would be better named `waitForGenesisBlock`, and checking true or false instead of defining it as a negative.